integration-tests/integration/coordinator/remove-involved-staff.spec.js

```diff
@@ -106,6 +105,10 @@ context('A use of force coordinator can remove involved staff', () => {
           { username: 'TEST_USER name', link: 'View statement', isOverdue: false, isUnverified: false },
         ])
       )
+
+    cy.task('getReportCount', [ReportStatus.SUBMITTED.value, ReportStatus.IN_PROGRESS.value]).then(count =>
```

Comment:
`seedReport` takes a ReportStatus and unwraps there rather than passing raw values through like this which is preferable but not the end of the world I guess.

Reply:
Yes. I did it this way to make the method generic so it could be used with different args. 
In `remove-involved-staff.spec.js` i've passed in SUBMITTED and IN_PROGRESS
but in `add-involved-staff.spec.js`, have passed in COMPLETE